taxi_service/settings.py

```diff
@@ -45,6 +45,9 @@
     "debug_toolbar",
     "crispy_forms",
     "taxi",
+    ...,
+   "crispy_bootstrap4",
+   "crispy_forms",
```

Comment:
The `INSTALLED_APPS` list contains duplicate entries for `crispy_forms`. Remove the duplicate to avoid redundancy.